### PR TITLE
Format files using the RuboCop linter.

### PR DESCRIPTION
**Pull request by AdinaBudriga**
These files were changed automatically by running rubocop --auto-correct.

### CODE REVIEW

File: utilities/stats.rb [hunk 1]
```diff
@@ -19,10 +19,10 @@ def initialize(stats = {})
     STATS.keys.each { |key| send "#{key}=", (stats[key] || 0) }
   end
 
-  def change_stats(other, sign)
+  def change_stats(_other, sign)
```
Comment (catalindrg) [inline utilities/stats.rb]: see above

File: character/hero.rb [hunk 2]
```diff
@@ -60,7 +60,7 @@ def change_wearable(wearable, free = true)
       @stats.change_stats(equipment.boots.stats, :+)
       @stats.coins -= @equipment.boots.stats.coins
     end
-  #  @stats.coins -= wearable.stats.coins unless free
+    #  @stats.coins -= wearable.stats.coins unless free
```
[the remark this answers is not in the document]
Comment (AdinaBudriga) [reply character/hero.rb]: Done. See commit [4383558](https://github.com/narcisgiorgianchitescu/RPG_Game2.0/pull/83/commits/438355898d3dff84351d683e1ecefd27636500fa).

File: utilities/stats.rb (hunk 1 above)
Comment (AdinaBudriga) [reply utilities/stats.rb]: See commit [4383558](https://github.com/narcisgiorgianchitescu/RPG_Game2.0/pull/83/commits/438355898d3dff84351d683e1ecefd27636500fa).

Comment (AdinaBudriga) [reply utilities/stats.rb]: I think the linter just didn't catch that the 'other' variable is actually expended before the eval call so it marked (and renamed it) as unused. I guess I'll just go back again to 'other' (so not _other).

**Pull request merged**